src/lib/file.ts

```diff
@@ -0,0 +1,19 @@
+export async function parseCSVFormData(file: FormDataEntryValue): Promise<string[]> {
+  if (!(file instanceof Blob)) {
+    throw new Error('Unable to process file');
+  }
+
+  const stream = file.stream() as unknown as AsyncIterable<any>;
+
+  if (file.type !== 'text/csv') {
+    throw new Error('Only Csv file is supported');
```

Comment:
Sinc eit is an acronym, it should be:

 ```suggestion
    throw new Error('Only CSV file is supported');
```

In addition the message could be improved to:

 ```suggestion
    throw new Error('The file is of the incorret expected type (CSV)');
```

Reply:
Fixed in https://github.com/nimblehq/ic-nextjs-prabesh-dahal/pull/59/commits/60c2b4b73860a885a6a8b0065d51e0ec37930de6